tests/phpunit/elementor/core/files/css/test-base.php

```diff
@@ -0,0 +1,91 @@
+<?php
+namespace Elementor\Tests\Phpunit\Elementor\Core\Files\Css;
+
+use ElementorEditorTesting\Elementor_Test_Base;
+
+/**
+ * Test the CSS Base class
+ */
+class Test_Base extends Elementor_Test_Base {
+
+	/**
+	 * @var \Elementor\Core\Files\CSS\Post
+	 */
+	private static $css_base_class;
+	/**
+	 * @var array
+	 */
+	private static $mock_control;
+	/**
+	 * @var array[]
+	 */
+	private static $mock_controls_stack;
+
+	public static function setUpBeforeClass() {
+		// The CSS Base class is abstract, so it can't be instantiated. The inheriting Post class is used instead.
+		self::$css_base_class = new \Elementor\Core\Files\CSS\Post( 0 );
```

Comment:
It's a better practice to have a fresh instance for each test instead of reusing the same one

Reply:
Done